packages/cysync-core/src/dialogs/Inheritance/EditEncryptedMessage/Dialogs/Decryption.tsx

```diff
@@ -0,0 +1,108 @@
+import { SignTransactionDeviceEvent } from '@cypherock/coin-support-interfaces';
+import {
+  ArrowRightIcon,
+  Check,
+  LeanBox,
+  LeanBoxContainer,
+  LeanBoxProps,
+  MessageBox,
+  tapAnyCardDeviceAnimation2DVideo,
+  Throbber,
+  Typography,
+} from '@cypherock/cysync-ui';
+import React, { useEffect } from 'react';
+
+import { selectLanguage, useAppSelector } from '~/store';
+
+import { useInheritanceEditEncryptedMessageDialog } from '../context';
+import { Layout } from '../Layout';
+import { LoaderDialog } from '~/components';
+import ReactPlayer from 'react-player';
+
+const checkIconComponent = <Check width={15} height={12} />;
+const throbberComponent = <Throbber size={15} strokeWidth={2} />;
+const rightArrowIcon = <ArrowRightIcon />;
+
+export const Decryption = () => {
+  const lang = useAppSelector(selectLanguage);
+
+  const strings =
+    lang.strings.dialogs.inheritanceEditEncryptedMessage.decryption.wallet;
+
+  const { onNext } = useInheritanceEditEncryptedMessageDialog();
+  const loading = false;
+
+  const deviceEvents: Record<number, boolean | undefined> = {
+    0: true,
+  };
+
+  const getDeviceEventIcon = (
+    loadingEvent: SignTransactionDeviceEvent,
+    completedEvent: SignTransactionDeviceEvent,
+  ) => {
+    if (deviceEvents[completedEvent]) return checkIconComponent;
+    if (deviceEvents[loadingEvent]) return throbberComponent;
+
+    return undefined;
+  };
+
+  const actionsList = React.useMemo<LeanBoxProps[]>(() => {
+    const actions: LeanBoxProps[] = [
+      {
+        id: '1',
+        text: strings.actions.tapCard,
+        leftImage: rightArrowIcon,
+        rightImage: getDeviceEventIcon(0, 1),
+      },
+    ];
+
+    return actions;
+  }, []);
+
+  useEffect(() => {
+    const timeout = setTimeout(() => {
+      onNext();
+    }, 5000);
+
+    return () => clearTimeout(timeout);
+  }, []);
+
+  if (loading) {
+    return (
+      <LoaderDialog
+        title={strings.decryption.title}
+        subtext={strings.decryption.subTitle}
+      />
+    );
+  }
+
+  return (
+    <Layout>
+      <ReactPlayer
```

Comment:
Try to use `Video` component
```suggestion
      <Video
```